wagesproject.sh

```diff
@@ -0,0 +1,32 @@
+#write a file with each uniqe gender/yearsExperience combo in wages.csv
```

Comment:
1.	1
2.	You correctly answer the first two parts of the question but the 3rd part is incorrect. You list 0 females in the two 10 earners and there should be 2. They way you’ve sorted the data by gender and then wage creates output with all females first and all males second so when you call tail it’s going to grab the last 10 lines which are all males. Sorting by just wage and then taking the head or tail depending on how your sort just the wages will allow you to get the genders of the top 10 earners: cat wages.csv | grep -v "gender" | sort -t , -k 4 -n -r | head -n 10 | grep "female" | cut -d , -f 1,2
a.	0.9
3.	1

2.9/3